rest/resource-server/src/docs/asciidoc/projects.adoc

```diff
@@ -247,3 +247,16 @@ include::{snippets}/should_document_link_releases/curl-request.adoc[]
 ===== Example response
 include::{snippets}/should_document_link_releases/http-response.adoc[]
 
+[[resources-project-get-download-licenseinfo]]
+==== Download License Info
+
+A `GET` request is used to download license info for the project.
+
+===== Example request
+include::{snippets}/should_document_get_download_license_info/curl-request.adoc[]
+
+===== Example request parameter
```

Comment:
Could you document "Request parameters" before "Example request" as it is done for other rest end point documentation.

Reply:
Updated the order.